Library/Homebrew/cmd/test-bot.rb

```diff
@@ -427,7 +427,7 @@ def formula(formula_name)
         "#{@tap}/#{formula_name}"
       end
 
-      test "brew", "uses", "--recursive", canonical_formula_name
+      test "brew", "uses", canonical_formula_name
```

Comment:
But, as this particular call is fairly slow, it could be removed in favour of the call below.